i3lock-fancy-rapid.c

```diff
@@ -93,12 +93,67 @@ void box_blur(unsigned char *dest, unsigned char *src, int height, int width,
 
 }
 
+void pixelate(unsigned char *dest, unsigned char *src, int height,
+                   int width, int radius)
+{
+#pragma omp parallel for
+    for (int i = 0; i < height; i += radius) {
+#pragma omp parallel for
```

Comment:
Nested `#pragma omp parallel for` doesn't help, it'll create `N^2` threads and saturate the OS scheduler.
```suggestion

```